### ISSUE TITLE
Can't revert or visit a previous version

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#158]
When I click on Visit, I see this:
![image](https://user-images.githubusercontent.com/656259/104127714-9049ac80-5318-11eb-8d7e-b362eb7ad456.png)
And after a couple of minutes I see this:
![image](https://user-images.githubusercontent.com/656259/104127743-b40cf280-5318-11eb-8d18-8b2fa42f34e0.png)

**Comment by rickmcgeer** [comment]
This is the error in the console:
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Object.EK [as knownModuleNames] (__root_module__-fd8ea0c8.js:385)
    at Cs.ensureCache (__root_module__-fd8ea0c8.js:2938)
    at Cs.getModuleIdsForPackageURL (__root_module__-fd8ea0c8.js:2939)
    at sj.modules (__root_module__-fd8ea0c8.js:2767)
    at sj.remove (__root_module__-fd8ea0c8.js:2776)
    at sj.reload (__root_module__-fd8ea0c8.js:2777)
    at $jscomp.generator.Engine_.program_ (__root_module__-fd8ea0c8.js:403)
    at $jscomp.generator.Engine_.nextStep_ (load.js:46)
    at $jscomp.generator.Engine_.next_ (load.js:42)
(anonymous) @ __root_module__-fd8ea0c8.js:5060
$jscomp.generator.Engine_.nextStep_ @ load.js:46
$jscomp.generator.Engine_.throw_ @ load.js:44
throw @ load.js:47
Kb @ load.js:48
Promise.then (async)
Td @ load.js:48
(anonymous) @ load.js:48
$jscomp.asyncExecutePromiseGenerator @ load.js:48
$jscomp.asyncExecutePromiseGeneratorProgram @ load.js:48
visitSelectedCommit @ __root_module__-fd8ea0c8.js:5059
onMouseDown @ __root_module__-fd8ea0c8.js:5059
dispatchEvent @ __root_module__-fd8ea0c8.js:2696
(anonymous) @ __root_module__-fd8ea0c8.js:2698
dispatchDOMEvent @ __root_module__-fd8ea0c8.js:2698
v @ __root_module__-fd8ea0c8.js:2684
__root_module__-fd8ea0c8.js:4003 Uncaught (in promise) TypeError: f.showErrorFor is not a function
    at d.hasOwnProperty.d.World.d.World.showError (__root_module__-fd8ea0c8.js:4003)
    at $jscomp.generator.Engine_.program_ (__root_module__-fd8ea0c8.js:5060)
    at $jscomp.generator.Engine_.nextStep_ (load.js:46)
    at $jscomp.generator.Engine_.throw_ (load.js:44)
    at $jscomp.generator.Generator_.throw (load.js:47)
    at Kb (load.js:48)

**Comment by merryman** [comment]
Yep, this is definitely a bug. We are trying to load the world directly within the dashboard which can never work. I will fix.
